lib/mix/lib/mix/tasks/compile.app.ex

```diff
@@ -18,10 +18,12 @@ defmodule Mix.Tasks.Compile.App do
   The most commonly used options are:
 
     * `:extra_applications` - a list of Erlang/Elixir applications
-      that you want started before your application. For example,
-      Elixir's `:logger` or Erlang's `:crypto`. Mix guarantees
-      that any application given here and all of your runtime
-      dependencies are started before your application starts.
+      your application depends on which are not included in `deps/0`.
```

Comment:
What do you think about?

> * `:extra_applications` - a list of Erlang/Elixir applications your application depends on which are not included in `:deps` (usually defined in `deps/0` in your `mix.exs`).

The main idea is to reflect that `deps/0` is a convention.

Reply:
Good point, that is more precise. Going to amend.